### PR TITLE
Fixes: Syringes causing welder tanks to explode

### LINKED ISSUE #618: Syringe gun shot blows up a fueltank
### Expected Behaviour

I shoot a fueltank with a syringe -- it does not blow up.
### Actual Behaviour

It goes boom.
### Steps to reproduce the problem
1. Take a syringe gun
2. Load any syringe inside
3. Pew pew

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

Fixes #618 where syringes blew up welder tanks.

### CODE REVIEW

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -104,7 +104,8 @@
 	..()
 	if(istype(Proj ,/obj/item/projectile/beam)||istype(Proj,/obj/item/projectile/bullet))
 		if((Proj.damage_type == BURN) || (Proj.damage_type == BRUTE))
-			if(Proj.nodamage)
+			if(Proj.nodamage || istype(Proj,/obj/item/projectile/bullet/dart/syringe))
```
Comment (Cruix) [inline code/modules/reagents/reagent_dispenser.dm]: No, specific checks are bad, change something about syringes or add a new var to projectiles. Or perhaps make the tank only explode if more than a certain amount of damage is done, so foam darts will pens in them will also not blow up welder tanks.

Comment (Super3222) [inline code/modules/reagents/reagent_dispenser.dm]: Alright, I'll make it more universal.

**Comment by Super3222** [comment]
Actually, give me a second here.

**Comment by Super3222** [comment]
what

**Pull request closed without merging**